lib/dirac.cpp

```diff
@@ -254,55 +254,274 @@ namespace quda {
 
     return nullptr;
   }
-  
-  // Count the number of stencil applications per dslash application.
-  int Dirac::getStencilSteps() const
+
+  bool Dirac::is_wilson_type(QudaDiracType type)
+  {
+    switch (type) {
+    case QUDA_WILSON_DIRAC:
+    case QUDA_WILSONPC_DIRAC:
+    case QUDA_CLOVER_DIRAC:
+    case QUDA_CLOVERPC_DIRAC:
+    case QUDA_CLOVER_HASENBUSCH_TWIST_DIRAC:
+    case QUDA_CLOVER_HASENBUSCH_TWISTPC_DIRAC:
+    case QUDA_TWISTED_CLOVER_DIRAC:
+    case QUDA_TWISTED_CLOVERPC_DIRAC:
+    case QUDA_TWISTED_MASS_DIRAC:
+    case QUDA_TWISTED_MASSPC_DIRAC: return true; break;
```

Comment:
the `break` after `return here is redundant (and similar below).  Not that it's a big deal.

Reply:
Discussed offline, since this "does no harm" I'll put a fix into one of the other branches I'm working on.

Reply:
jk, done: [919a6ef](https://github.com/lattice/quda/pull/1488/commits/919a6eff37660a7614fcc8b1e8f44cfcb3918779)